behat/tests/bootstrap/v2/reporting/sections/ReportingSectionsFeatureContext.php

```diff
@@ -64,7 +65,8 @@ public function nextSectionShouldBe(string $sectionName)
      */
     public function iNavigateBackToReportSection()
     {
-        $this->clickLink('Deputy report overview');
+        $linkText = $this->reportUrlPrefix == 'ndr' ? 'New deputy report overview' : 'Deputy report overview';
```

Comment:
If we are doing this a lot, it might be better to just start adding `data-action="report.overview" `style hooks onto the links.

Reply:
Done that